src/pkg/member/models/member.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package models
+
+import (
+	"github.com/goharbor/harbor/src/pkg/user/models"
```

Comment:
Let's avoid the importing for other pkg packages, this causes import cycle easily 

I check the use of `models.User`, seems only the `UserID` and `GroupID` are used, wrap them in a new model or use them directly

Reply:
Updated